README.md

```diff
@@ -1,11 +1,12 @@
 [![Logotype](https://github.com/schemaspy/schemaspy/blob/master/docs/source/_static/schemaspy_logo.png)](https://schemaspy.org/)
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.schemaspy/schemaspy/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.schemaspy/schemaspy/)
```

Comment:
How come you went with this heroku app, instead of [shields.io](https://shields.io/)?

Reply:
shields.io is my go-to badges generator. However, AFAIK they don't have Maven Central badges with automatic redirect (i.e. only the badge, not the Central link).